### ISSUE TITLE
Dot converted to comma

### ISSUE THREAD

**Issue by ryuran** [pocketjoso/penthouse#194]
Hello

My original css have some float value with css valid dot.
```css
font-size: 62.5%;
```
But penthouse transform this dot to comma
```css
font-size: 62,5%;
```
and the value become invalid

# EDIT:
When `strict: false` and ASTexplorer fail, penthouse run chrome CSS nomalizer then this one convert dots to comma.

## Test case: 
```css
body {
  /* this data URI fail on ASTexplorer*/
  background-image: url("data:image/svg+xml;utf8,%3Csvg%20version='1.1'%20xmlns='http://www.w3.org/2000/svg'%20xmlns:xlink='http://www.w3.org/1999/xlink'%20viewBox='0%200%2010%2010'%3E%3Cstyle%3Epath%7Bfill:%20rgb(0,0,0);%7D%3C/style%3E%3C/svg%3E");
  /* this font-size value is transformed to 62,5% by chrome css normalizer */
  font-size: 62.5%;
}
```

**Comment by pocketjoso** [comment]
Are you sure about this coming from Penthouse? The only transformation it does of your original css is when your css is invalid, in which case it uses Chrome to normalize the css.. To see if this was what happened, can you please re-run penthouse with `strict: true`? It will then throw if you have errors, instead of trying to normalise the css.

**Comment by ryuran** [comment]
ASTexplorer return an error on this line, but it seems correct. (svg as data uri)

http://astexplorer.net/#/gist/424af2e86d4ec2f48ea91d0b3a16b2d0/ea6dda2103bf9a95adc9848c781f0bd7cf68565c

I have the same error with `strict: true`

I don’t know how to fix it.

**Comment by pocketjoso** [comment]
Okay, so two separate issues:
* The ast parser fails to parse your `;` in the data-uri: could you please open up a separate issues for this?

* the one regarding the chrome normalisation error.
Can you add a minimal reproducible test case? What css is needed to reproduce the error?

**Comment by ryuran** [comment]
On this project? @pocketjoso

**Comment by pocketjoso** [comment]
@ryuran 
Let's keep this issue for what the chrome normalisation error (`,` into `.`).
To proceed debugging that, can you please provide a minimal reproducible test case - some (reduced) css that can be used to reproduce the error?

For the ast issue, yeah I think let's keep it here to keep track of it. There won't be any quick resolution as it would most likely require changing AST parser. If you're interested in having something happen before then, feel free to look into a fix on the [current css AST parser fork](https://github.com/pocketjoso/css).

**Comment by ryuran** [comment]
I edited first post.

**Comment by pocketjoso** [comment]
Thanks for the update, I will look into this problem as soon as I find time - hopefully the upcoming weekend!

**Comment by pocketjoso** [comment]
Should be fixed by 1.2.0 - please give it a go:
https://github.com/pocketjoso/penthouse/releases/tag/v1.2.0
